src/toTransform.js

```diff
@@ -39,36 +39,77 @@
 
   function convertPath (properties) {
     var offsetPath = internalScope.offsetPathParse(properties['offset-path']);
-    if (!offsetPath || offsetPath.type !== 'path') {
+
+    if (!offsetPath) {
       return null;
     }
 
+    if (offsetPath.type === 'path') {
+      return convertPathString(properties);
+    }
+
+    if (offsetPath.type === 'ray') {
+      return convertRayString(properties);
+    }
+  }
+
+  function getOffsetDistanceLength (offsetDistance, pathElement) {
+    if (offsetDistance.unit === '%') {
+      return Number(offsetDistance.value) * pathElement.getTotalLength() / 100;
+    } else {
+      return Number(offsetDistance.value);
```

Comment:
Optional: We shouldn't have to be converting to Number here, that should happen in the parse method.